profiles/config.py

```diff
@@ -23,24 +23,29 @@ def __init__(self, orcid: Dict[str, str], db: str, logging: Dict[str, str],
 
 class DevConfig(Config):
     DEBUG = True
+    SERVER_NAME = 'localhost'
     name = 'dev'
 
 
 class CiConfig(DevConfig):
     TESTING = True
+    SERVER_NAME = 'ci--profiles.elifesciences.org'
     name = 'ci'
 
 
 class ProdConfig(Config):
     name = 'prod'
+    SERVER_NAME = 'prod--profiles.elifesciences.org'
```

Comment:
This should probably be configuration? /cc @giorgiosironi

Reply:
Might want to set `PREFERRED_URL_SCHEME` at the same time? (Default's `http`, but we use `https` everywhere outside of Vagrant.)